config.js

```diff
@@ -2,6 +2,8 @@ let config = {
   enableUpdateDescribeGraph: false,
   postRateLimit: 100,
   rateLimit: 200,
+  defaultForwardReserveFee: 0.01,
+  defaultIntraHubFee: 0.003,
```

Comment:
The naming here seems inconsistent. `forwardReserveFee` and `intraHubFee` I think would be appropriate without the `default`.

Reply:
the variables at config represents default values at invocation that can later be adjusted at runtime, it also represents what was hard-coded as defaults, therefore chosen for clarity, i do not mind if You call them Bill & Linda ...